src/backend/distributed/commands/foreign_constraint.c

```diff
@@ -536,6 +535,31 @@ GetTableForeignConstraintCommands(Oid relationId)
 }
 
 
+/*
+ * get_relation_constraint_oid_compat returns OID of the constraint represented
```

Comment:
Comment should be updated to reflect that we can still retrieve the OID directly in pg11

Reply:
addressed